### ISSUE TITLE
Pretrained Model

### ISSUE THREAD

**Issue by jiachens** [pbias/lunet#2]
Nice work!

Any possibility to release the pre-trained model which can reproduce the results in your paper?

**Comment by pbias** [comment]
Good morning,

Unfortunetely, I'm not planning on publishing the trained model. However,
you can access the same results by training it following the protocol (with
the slight upper/lower margin that will be induced by the random shuffling
of the training set during the training).

Pierre.

Le dim. 1 déc. 2019 à 23:23, Jiachen Sun <notifications@github.com> a
écrit :

**Comment by jiachens** [comment]
Thanks for replying.

BTW, your RIU-Net paper and LU-Net paper use the same figure (example) to illustrate the performance in the first page, but the groudtruth seems different, not sure whether it is a good way to publish a paper.

**Comment by pbias** [comment]
Hi there,

The depth colormap (2D) and point of view (3D) of the example figure in
page 1 are indeed different as for better reading in the LU-Net paper, as
it was intended to be submitted, and was accepted, to the CVRSUAD workshop
of ICCV.

Not sure if I correctly understand your remark about the good way to
publish a paper, though...

Le mar. 3 déc. 2019 à 21:14, Jiachen Sun <notifications@github.com> a
écrit :
